data/campaigns/Descent_Into_Darkness/scenarios/06_Return_to_Parthyn.cfg

```diff
@@ -496,21 +497,21 @@
         {VARIABLE_OP malin.experience add 12}
         [floating_text]
             x,y=$malin.x,$malin.y
-            text="<span color='#800080'>" + "+12 exp" + "</span>"
+            text=_ "<span color='#800080'>" + "+12 exp" + "</span>"
```

Comment:
Again, I think the underscore should instead go after the first `+`.

Reply:
Incidentally, this seems like a situation that wmllint _might_ be able to detect automatically? If a line consists of three strings concatenated by `+` where the first and third string both begin with `<` and end with `>`, then it's probably this situation. In fact, maybe that could even be extended to an arbitrary sequence of strings concatenated by `+`.

@Elvish-Hunter Any thoughts on this idea?

Reply:
Sounds too specific. How often would this rule be relevant?